templates/web/base/maps/openlayers.html

```diff
@@ -31,6 +31,11 @@
     <div id="map">
       [% INCLUDE 'maps/noscript_map.html' js = 1 %]
     </div>
+  [% IF c.req.params.no_pins %]
```

Comment:
I guess it's possible you might want to turn off pins on a `/reports` or `/report` page, but I think we probably only want this on an `/around` page :)